interview-questions.md

```diff
@@ -4,39 +4,39 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer:you would use it to call the base classe's constructor
 
   Researched answer:
 
 
 
 2. What is a virtual DOM?
 
-  Your answer:
+  Your answer:its the representation of the UI
 
   Researched answer:
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to display "Hello World" in the browser?
 
-  Your answer:
+  Your answer: const = props => React.Element
 
   Researched answer:
 
 
 
 4. What is JSX?
 
-  Your answer:
+  Your answer:it allows you to use html and JavaScript
 
   Researched answer:
 
 
 
 5. What is state in React?
 
-  Your answer:
+  Your answer: state is where you store property values that belong to the components
```

Comment:
Pretty good answer! I'd say it would be worth noting that its for dynamic use as well.